### PR TITLE
#17885 upgrade ES version

### PR DESCRIPTION
**Pull request by dsilvam**

### CODE REVIEW

File: dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java [hunk 1]
```diff
@@ -63,33 +68,44 @@ public void findAllCurrent () throws DotDataException, DotSecurityException {
...
+                        if (i % 2 == 0) {
+                            Sneaky.sneaked(() -> APILocator.getContentletAPI()
+                                    .publish(newContent, user, false));
+                        }
+                        newContentlets.add(newContent);
+                    }
+            );
+
+            //Getting all contentlets live/working contentlets
+            List<Contentlet> contentlets = contentletFactory.findAllCurrent(0, 10);
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java]: Codacy Issue found: [Local variable 'contentlets' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4884522)

File: dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java [hunk 2]
```diff
@@ -63,33 +68,44 @@ public void findAllCurrent () throws DotDataException, DotSecurityException {
...
+            assertTrue(contentlets != null && !contentlets.isEmpty());
+            assertTrue(contentlets.size() >= 10);
+
+            //Search for one of the objects we found
+            String inode = contentlets.get(0).getInode();
+            Contentlet contentlet = contentletFactory.find(inode);
+
+            //Validations
+            assertTrue(
+                    contentlet != null && (contentlet.getInode() != null && !contentlet.getInode()
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java]: Codacy Issue found: [Useless parentheses.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4884522)

File: dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java [hunk 3]
```diff
@@ -63,33 +68,44 @@ public void findAllCurrent () throws DotDataException, DotSecurityException {
...
+
+            //Getting all contentlets live/working contentlets
+            List<Contentlet> contentlets = contentletFactory.findAllCurrent(0, 10);
+
+            //Validations
+            assertTrue(contentlets != null && !contentlets.isEmpty());
+            assertTrue(contentlets.size() >= 10);
+
+            //Search for one of the objects we found
+            String inode = contentlets.get(0).getInode();
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java]: Codacy Issue found: [Local variable 'inode' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4884522)

File: dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java [hunk 4]
```diff
@@ -63,33 +70,46 @@ public void findAllCurrent () throws DotDataException, DotSecurityException {
...
+            Contentlet contentlet = contentletFactory.find(inode);
+
+            //Validations
+            assertTrue(
+                    contentlet != null && (contentlet.getInode() != null && !contentlet.getInode()
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java]: Codacy Issue found: [Useless parentheses.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4884522)

File: dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java [hunk 5]
```diff
@@ -63,33 +70,46 @@ public void findAllCurrent () throws DotDataException, DotSecurityException {
...
+            List<Contentlet> contentlets = contentletFactory.findAllCurrent(0, 10);
+
+            // filter out null records
+            contentlets = contentlets.stream().filter(Objects::nonNull).collect(Collectors.toList());
+            //Validations
+            assertTrue(contentlets != null && !contentlets.isEmpty());
+            assertTrue(contentlets.size() >= 10);
+
+            //Search for one of the objects we found
+            String inode = contentlets.get(0).getInode();
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletFactoryTest.java]: Codacy Issue found: [Local variable 'inode' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4884522)

**Review by nollymar — APPROVED** [review]

**Review by freddyucv — APPROVED** [review]

**Pull request merged**